source/vscode/changelog.md

```diff
@@ -1,5 +1,44 @@
 # QDK Changelog
 
+## v1.21.0
+
+Below are some of the highlights for the 1.21 release of the QDK.
+
+### QDK Python package
+
+With this release we are also publishing a `qdk` package to PyPI (see <https://pypi.org/project/qdk/>). This is still in the 'preview' stage as we lock down the API, but the goal is that going forward the QDK will be installed in Python via `pip install qdk`, with any optional extras needed (e.g. `pip install "qdk[jupyter,azure,qiskit]"` to add the Jupyter Notebooks, Azure Quantum, and Qiskit integration). Once installed, import from the necessary submodules (e.g. `from qdk.openqasm import compile`)
+
+Please give it a try and open an issue if you have any feedback.
+
+### Complex literals
+
+The Q\# language added support for complex literals. For example,
+
+```qsharp
+function GetComplex() : Complex {
+    3. + 4.i
+}
+```
+
+This can be used in arithmetic expressions (e.g., `(2.0 + 3.0i) * (4.0 + 5.0i)`) and passed to and from Python.
```

Comment:
I think it's worth adjusting this slightly to clarify that arithmetic support works with any complex, not just complex literals (GitHub doesn't like me trying to put a code sample into a suggestion block, so just writing it out below):

> Additionally, `Complex` values can now be used in arithmetic expressions directly:
> ```qsharp
> let x = 2.0 + 3.0i;
> let y = x + (4.0 - 5.0i);
> ```